test/bin/test-browser.js

```diff
@@ -0,0 +1,240 @@
+const { spawn } = require('child_process')
+const tmp = require('tmp-promise')
+const dht = require('@hyperswarm/dht')
+const HyperspaceServer = require('hyperspace/server')
+const HyperswarmServer = require('hyperswarm-web/server')
+const webnet = require('webnet')
+const http = require('http')
+const tapeRun = require('tape-run')
+const path = require('path')
+const { createWriteStream, createReadStream } = require('node:fs')
```

Comment:
Not sure what the `node:` thing is supposed to mean so I removed it in `master`

Reply:
Thank you for removing it, it is unnecessary. yhe "node:" prefix is the new way to make sure to differentiate between internal libraries and the ones on npm.